lib/api/bucketDelete.js

```diff
@@ -31,17 +31,17 @@ export default function bucketDelete(authInfo, request, log, cb) {
         log,
```

Comment:
You can remove the log here since we have it as a separate param in the `metadataValidateBucketAndObj` function. If you don't want to remove the log from all the `metadataValParams` constants, you can change the `metadataValidateBucketAndObj` function parameters to expect it in the params object (unfortunately, you'll still have to modify the existing `metadataValidateBucketAndObj` references that were used for the versioning apis)